runtime/compiler/z/runtime/PicBuilder.m4

```diff
@@ -1193,7 +1193,7 @@ ifdef([OMR_GC_COMPRESSED_POINTERS],[dnl
 
     MTComputeStaticAddress(AddressLoad,Obj)
 ifdef([OMR_GC_COMPRESSED_POINTERS],[dnl
-    LLGF    r3,0(,r2)                        # load comprssed val
+    LLGF    r3,0(,r2)                        # load compressed val
     SLLG    r3,r3,0(r1)                       # decompress
```

Comment:
This whitespace isn't my fault (I'm pretty sure I've broken some others)

Reply:
Should be fine. Thanks for your contribution.